src/ParametricUtils.jl

```diff
@@ -195,6 +220,27 @@ function calcFactorMahalanobisDict(fg)
   return calcFactors
 end
 
+# new experimental Manifold ProductRepr version
+function _totalCost(M, cfvec::Vector{<:CalcFactorMahalanobis}, labeldict, points)
+  #
+  obj = 0
+
+  for cfp in cfvec
+
+      varOrder = getindex.(Ref(labeldict), cfp.varOrder)
```

Comment:
again, julia is already pass by reference, so adding a second tier Ref over a dict ref seems weird to me

Reply:
Again for broadcasting.